src/Common/src/TypeSystem/Ecma/EcmaType.cs

```diff
@@ -537,6 +537,85 @@ private MarshalAsDescriptor GetMarshalAsDescriptor(FieldDefinition fieldDefiniti
             return null;
         }
 
+        public override PInvokeAttributes GetDelegatePInvokeAttributes()
```

Comment:
This might make more sense as a helper / extension method rather than member function in EcmaType. And probably should not be an override either.